bucket/winspy.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "1.0.3",
+    "description": "Window Information Tool",
+    "homepage": "https://sourceforge.net/projects/winspyex/",
+    "license": "Unknown",
+    "url": "https://downloads.sourceforge.net/project/winspyex/1.0.3/WinSpy-1.0.3.7z#/dl.7z",
+    "hash": "sha1:5ea58ded91f37fcb3d58fcaab0aaed1e723e9483",
+    "extract_dir": "WinSpy",
+    "architecture": {
+        "64bit": {
+            "shortcuts": [
+                [
+                    "WinSpy64.exe",
+                    "WinSpy"
+                ]
+            ]
+        },
+        "32bit": {
+            "shortcuts": [
+                [
+                    "WinSpy32.exe",
+                    "WinSpy"
+                ]
+            ]
+        }
+    },
+    "pre_install": "$manifest.persist | ForEach-Object { if (-not (Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -Force -ItemType \"file\" | Out-Null } }",
+    "persist": "WinSpy.ini",
+    "checkver": {
+        "regex": "Download WinSpy-([\\d.]+).7z from SourceForge"
+    },
+    "autoupdate": {
+        "url": "https://downloads.sourceforge.net/project/winspyex/$version/WinSpy-$version.7z#/dl.7z"
```

```suggestion
        "url": "https://downloads.sourceforge.net/project/winspyex/$version/WinSpy-$version.7z#/dl.7z"
```
`#/dl.7z` (rename the file as "dl.7z") is not needed because the file is already a 7-zip archive.